tardis/simulation/tests/test_simulation.py

```diff
@@ -61,3 +73,15 @@ def test_packet_output(simulation_one_loop, simulation_compare_data):
     assert_quantity_allclose(simulation_one_loop.runner.output_energy,
                         simulation_compare_data['test1/output_energy'] * u.Unit('erg'),
                         atol=0.0 * u.Unit('erg'))
+
+
+def test_make_source_function(simulation_one_loop,raw_model):
+    # For now just test that the method can be called without exceptions
+    succes = False
```

Comment:
You don't need these exception guards. You can simply run the function. If it throws an error, the test fails. If you expect an error, you can use `with pytest.raise(TypeError):` which will cause the test to fail if that exception was NOT raised.